src/main/java/module/tsforecast/ForecastCurve.java

```diff
@@ -145,106 +159,158 @@ else if (point2.m_dSpirit < 0.0D)
 		}
 	}
 
-	// -- protected
-	// ----------------------------------------------------------------------
+	/**
+	 *  Calculate the team spirit after changing the training intensity
+	 * 	// HT-Tools Hattrick Manager Assistant
+	 * 	//Copyright 2014-2015 Ventouris Anastasios
+	 * 	//Licensed under GPL v3
```

Comment:
This may be an issue: as I understand this, an application using GPLv3 code should be GPLv3 itself, due to the copyleft nature of GPL.  HO is licensed under LGPLv3.

Reply:
I agree with you: How should we deal with this - Is it a good idea to ask HT-Tassos for permission to transfer his code to Java?

Reply:
Since our HO source is not a library usable by other developers, I have never understood licensing under LGPL- Would we work around the problem if we switched to GPL for HO - Does that make sense?

Reply:
I think it would be simpler to ask whether HT-Tassos can re-license to LPGL.  I have seen projects in the past trying to change license, and they had to contact all contributors to get approval for re-licensing.

Reply:
Here is the answer of HT-Tasos to this:

> Hi there. I would say if you put a reference to my GitHub account, from your GitHub's main page and comment in the code as well it's fine. Is that ok for you ?

Reply:
Nice!  Thanks HT-Tasos!